### ISSUE TITLE
new domain gets stuck

### ISSUE THREAD

**Issue by ajgarrote** [destination-earth-digital-twins/DE330_Training_2024#13]
I'm trying to rerun the demo case but moving the center of the domain to a new location. I have created a new configuration file with these lines:

[general]
  case = "TEST_LAND@CASE_PREFIX@@CYCLE@_@CSC@"

[domain]
  name = "TEST_LAND"
  xlat0 = 38.52
  xlatcen = 38.52
  xlon0 = -1.68
  xloncen = -1.68`

but it gets stuck at this step (does not abort):
![screenshot_issue](https://github.com/destination-earth-digital-twins/DE330_Training_2024/assets/139155770/99575223-f0ab-447c-9965-494253168d13)
I have also increased the time for this task but it does not work.

**Comment by FlorianW-ZAMG** [comment]
The error is reproducible and occurs for both cy48t3 and cy46h1. For completeness, here are the complete domain specifications:  

[domain]    
  gridtype = "linear"  
  ilate = 11  
  ilone = 11
  name = "DEMO_60x80_2500m"
  nbzong = -1
  nbzonl = -1
  nimax = 49
  njmax = 69
  tstep = 75
  xbeta = 0.0
  xdx = 2500.0
  xdy = 2500.0
  xlat0 = 38.52
  xlatcen = 38.52
  xlon0 = -1.68
  xloncen = -1.68

The domain is completely over land and the job gets stuck in Part2. The last lines of the logfile are:
PSEA2(  3370)=   0.0000
 PSEA2(  3371)=   0.0000
 PSEA2(  3372)=   0.0000
 PSEA2(  3373)=   0.0000
 PSEA2(  3374)=   0.0000
 PSEA2(  3375)=   0.0000
 PSEA2(  3376)=   0.0000
 PSEA2(  3377)=   0.0000
 PSEA2(  3378)=   0.0000
 PSEA2(  3379)=   0.0000
 PSEA2(  3380)=   0.0000
 PSEA2(  3381)=   0.0000

The NODE.001_01 ends with:  

-----------------------------------------------------------------------
 ------ END OF SETUPS at level 0 ---------------------------------------
 -----------------------------------------------------------------------


 SETTING UP INITIAL CONDITIONS     9   923
 COMMON YOMCLI
 LIEEE= T LGLOBE= T NDATX= 360 NDATY= 180 NAEROF=   0 NPINT=  3 EDLON (dg)= 1.00 EDLAT (dg)= 1.00
 SVEG =  0.02 LZ0THER =  T SFCZ0 =  1.00 RSTR =  5.K  RSWR = 0.02 NSLICE =    1
 NILCLI1=    1 NILCLI2=  360 NJLCLI1=    1 NJLCLI2=  180
  HAF, HAF : CADRE : Const.Clim.Surfa
 GLOBALLY DOMINANT LAND USE IS            3
 NO MASK FOR BARE GROUND ALBEDO AND EMISSIVITY !

**Comment by PatrickSamuelsson** [comment]
I'm sorry, I have no great success in my investigation. I consider myself more talented in the SURFEX part (pgd, prep,...) and I'm not familiar with the e9xx parts of the system. I see that the message "NO MASK FOR BARE GROUND ALBEDO AND EMISSIVITY" probably has nothing(?) to do with the error itself since the error appears later in the process and the message just happened to be the last thing written to the log. My understanding is that the execution stops in ./aladin/c9xx/einter10.F90 between the write statements (see ~/deode_ecflow/jobout/CY48t3_AROME_DEMO_60x80_2500m/StaticData/E923Constant.1)
  WRITE(6,'('' SORRY : NEVER SECONDARY TYPE '')')
  WRITE(0,'('' EINTER10 : NEVER SECONDARY TYPE '')')
  WRITE(0,'(''     ---> CRASH AVOIDED BUT LOWER SECURITY LEVEL'')')
... which are written to the log and the write statement
WRITE(6,'(''----- fin de EINTER10 - OK!'')')
... which is never written to the log.

I guess we need help from Météo-France colleagues (e.g. via Eric Bazile) to solve this problem...

**Comment by ajgarrote** [comment]
I completely agree. This is a simple test, not a real case. I opened this discussion in case it might be useful. Thank you very much for your time and sorry for the disruption.
